packages/application-config/test/process-config.spec.js

```diff
@@ -486,6 +491,201 @@ describe('processing a config with environment variable placeholders', () => {
   });
 });
 
+describe('processing a config with intl variable placeholders', () => {
+  beforeEach(() => {
+    loadConfig.mockReturnValue(fixtureConfigIntlVariables);
+  });
+  it('should process the config and correctly parse the translations', () => {
+    const result = processConfig(createTestOptions());
+    expect(result).toEqual({
```

Comment:
I know snapshots have a bad rep nowadays, but `toMatchInlineSnapshot` does a quite decent job in cases like this one. =)

Reply:
Yeah inline snapshots are actually useful in such cases.